enterprise_access/apps/api_client/tests/test_license_manager_client.py

```diff
@@ -38,3 +46,267 @@ def test_get_subscription(self, mock_oauth_client):
+            "start_date": _days_from_now(-50, DATE_FORMAT_ISO_8601),
+            "expiration_date": _days_from_now(50, DATE_FORMAT_ISO_8601),
+            "enterprise_customer_uuid": self.mock_enterprise_customer_uuid,
+            "enterprise_catalog_uuid": self.mock_enterprise_catalog_uuid,
+            "is_active": True,
+            "is_current": True,
+            "is_revocation_cap_enabled": False,
+            "days_until_expiration": _days_from_now(50, DATE_FORMAT_ISO_8601),
+            "days_until_expiration_including_renewals": _days_from_now(50, DATE_FORMAT_ISO_8601),
+            "is_locked_for_renewal_processing": False,
+            "should_auto_apply_licenses": False,
+            "created": _days_from_now(-60, DATE_FORMAT_ISO_8601)
+        }
+        self.mock_customer_agreement = {
+            "uuid": self.mock_customer_agreement_uuid,
+            "enterprise_customer_uuid": self.mock_enterprise_customer_uuid,
+            "enterprise_customer_slug": "mock_slug",
+            "default_enterprise_catalog_uuid": self.mock_enterprise_catalog_uuid,
+            "disable_expiration_notifications": False,
+            "net_days_until_expiration": _days_from_now(50, DATE_FORMAT_ISO_8601),
+            "subscription_for_auto_applied_licenses": self.mock_subscription_plan_uuid,
+            "available_subscription_catalogs": [
+                self.mock_enterprise_catalog_uuid,
+            ],
+            "enable_auto_applied_subscriptions_with_universal_link": False,
+            "has_custom_license_expiration_messaging": False,
+            "modal_header_text": None,
+            "expired_subscription_modal_messaging": None,
+            "button_label_in_modal": None,
+            "url_for_button_in_modal": None
+        }
+        self.mock_learner_license_result_response = {
+            "uuid": self.mock_learner_license_uuid,
+            "status": LicenseStatuses.ACTIVATED,
+            "user_email": self.mock_user_email,
+            "activation_date": _days_from_now(-10, DATE_FORMAT_ISO_8601_MS),
+            "last_remind_date": _days_from_now(-5, DATE_FORMAT_ISO_8601_MS),
+            "subscription_plan_uuid": self.mock_subscription_plan_uuid,
+            "revoked_date": None,
+            "activation_key": self.mock_license_activation_key,
+            "subscription_plan": self.mock_subscription_plan
+        }
+        self.mock_learner_license_activation_response = {
+            "uuid": self.mock_learner_license_activation_uuid,
+            "status": LicenseStatuses.ACTIVATED,
+            "user_email": self.mock_user_email,
+            "activation_date": _days_from_now(-10, DATE_FORMAT_ISO_8601_MS),
+            "last_remind_date": _days_from_now(-5, DATE_FORMAT_ISO_8601_MS),
+            "subscription_plan_title": "mock_subscription_plan_title",
+            "subscription_plan_expiration_date": _days_from_now(50, DATE_FORMAT_ISO_8601),
+            "activation_link": self.faker.url()  # pylint: disable=no-member
```

Comment:
[nit / sanity check] Is this disabled pylint rule necessary?

Reply:
Yep, otherwise it throw's a linting error.